start.sh

```diff
@@ -158,6 +158,9 @@ EOI
     if yesno "Start the account service" Y; then
         PROFILES="${PROFILES} account"
     fi
+    if yesno "Start the mock webhook logger" Y; then
+        PROFILES="${PROFILES} webhook"
+    fi
```

Comment:
Added this to eliminate delays and error messages in `account-service-worker` due to a webhook URL being specified (mandatory), but nothing listening at the specified address.